### PR TITLE
Clamp audio samples to valid range to prevent NaN loss during training

### PR DESCRIPTION
**Pull request by shashank14k**
* I trimmed a large audio into segments, upon loading, one file had maximum sample value >1e29.
* This resulted in a NaN loss during training
* The audio is likely corrupted, and the loss should not equal NaN in most cases, but clamping the values within a range should handle edge cases

### CODE REVIEW

**Comment by shashank14k** [comment]
I could not attach the wav file as github does not support wav format. I could re-encode it to mp4, however rencoding discards such values.

**Comment by Plachtaa** [comment]
For project simplicity, dataloader should not be responsible for data corruption
